domain/shared/utils/dataTypeCheck.ts

```diff
@@ -0,0 +1,13 @@
+export const isJson = (data: any): boolean => {
+  let ret = true;
+  try {
+    JSON.parse(data);
+  } catch (e) {
+    ret = false;
+  }
+  return ret;
+};
+
+export default {
+  isJson,
+};
```

Comment:
why it need to wrapped in curly-braces ?

Reply:
so I can use like this
- dataTypeCheck.isJson()
- dataTypeCheck.isString() -- example
- dataTypeCheck.isArray() -- example